### PR TITLE
docs(terminus): replaced AppModule with the HealthModule

### PR DESCRIPTION
**Pull request by Scrip7**
- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type

- [x] Docs

## What is the current behavior?
The code block for writing the health controller exists, but there isn't a sample regarding its module.

## What is the new behavior?

I've added the code block.

## Does this PR introduce a breaking change?
- [x] No

### CODE REVIEW

**Review by micalevisk — CHANGES_REQUESTED** [review]
instead of adding another module, we could just add that `HealthController` to `AppModule`'s `controllers` array as it is used in the beginning of the page, just to make things more easy to follow.

I guess people can get confused because in this page we already have the `imports: [TerminusModule]`

**Comment by Scrip7** [comment]
Oh, sorry, my bad. @micalevisk 
I knew we could import that in the root `AppModule`, but I didn't notice we did that on top.

Wouldn't it be better if we guided people to create a separate `HealthModule` for that?

Cause I believe in a real-world scenario (at least in the handful scenarios I've had in the past);
- The `HealthController` is also dependent on the `HttpModule` from the `@nestjs/axios` and the configuration services to grab other microservices information
- Other modules shouldn't have access to the set of health indicators `@nestjs/terminus` provides
- It'll help organize health-checker-related things in its module rather than putting everything in the root module.

Please correct me if I'm wrong.

**Comment by micalevisk** [comment]
having `HealthModule` is far better and realistc for sure, but I think we're trying to make this example as simple as possible

actually, I don't know if we need to show the final version of the module because if we follow that page properly, we have ran `nest g controller health` and this will update the `AppModule` for us

:thinking:

**Comment by Scrip7** [comment]
Sure, I'll give it a try!
I was looking at the [samples](https://github.com/nestjs/terminus/tree/master/sample),
All of them have their dedicated health module 😄

File: content/recipes/terminus.md [hunk 1]
```diff
@@ -22,26 +22,32 @@ $ npm install --save @nestjs/terminus
...
-- `MemoryHealthIndicator`
-- `DiskHealthIndicator`
+-   `HttpHealthIndicator`
+-   `TypeOrmHealthIndicator`
+-   `MongooseHealthIndicator`
+-   `SequelizeHealthIndicator`
+-   `MicroserviceHealthIndicator`
+-   `GRPCHealthIndicator`
+-   `MemoryHealthIndicator`
+-   `DiskHealthIndicator`
```
Comment (micalevisk) [inline content/recipes/terminus.md:32]:
```suggestion
- `HttpHealthIndicator`
- `TypeOrmHealthIndicator`
- `MongooseHealthIndicator`
- `SequelizeHealthIndicator`
- `MicroserviceHealthIndicator`
- `GRPCHealthIndicator`
- `MemoryHealthIndicator`
- `DiskHealthIndicator`
```

**Review by micalevisk — APPROVED** [review]

File: content/recipes/terminus.md [hunk 2]
```diff
@@ -31,17 +31,23 @@ A health check represents a summary of **health indicators**. A health indicator
 - `MemoryHealthIndicator`
 - `DiskHealthIndicator`
 
-To get started with our first health check, we need to import the `TerminusModule` into our `AppModule`.
+To get started with our first health check, let's generate the `HealthModule` now using [Nest CLI](/cli/overview).
```
Comment (kamilmysliwiec) [inline content/recipes/terminus.md:34]: We can't assume everyone uses CLI. We should skip this sentence and consider adding it as a hint

Comment (Scrip7) [reply content/recipes/terminus.md:34]: Are you sure about that?
As Micael said earlier, the `nest g controller health` was there before any changes.

Should I still skip it?

**Comment by kamilmysliwiec** [comment]
@jmcdo29 can you double-check this when you have a chance? 🤞

**Review by jmcdo29 — APPROVED** [review]
LGTM

**Review by Tony133 — APPROVED** [review]

**Pull request merged**